src/watts/plugin_serpent.py

```diff
@@ -0,0 +1,74 @@
+# SPDX-FileCopyrightText: 2022 UChicago Argonne, LLC
+# SPDX-License-Identifier: MIT
+
+from pathlib import Path
+from typing import List, Optional
+
+from uncertainties import ufloat
+
+from .fileutils import PathLike
+from .plugin import TemplatePlugin
+from .results import Results
+
+
+class ResultsSerpent(Results):
+    """Serpent simulation results
+
+    Parameters
+    ----------
+    params
+        Parameters used to generate inputs
+    name
+        Name of workflow producing results
+    time
+        Time at which workflow was run
+    inputs
+        List of input files
+    outputs
+        List of output files
+
+    Attributes
+    ----------
+    keff
+        K-effective value
+    stdout
+        Standard output from Serpent run
+    """
```

Comment:
ResultsSerpent is empty right now - should ed comment out "Attributes" ?
In the future, consider using the SerpentTool to postprocess Serpent results and return them in Results.

Reply:
Ah, yes. `keff` should not be there, but `stdout` is inherited from the base `Results` class and can remain. I'll update this.